CorrelatorSharp/packages.config

```diff
@@ -1,4 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
By removing this a fresh checkout of the repo won't pull in the dependency on build unless I am missing something?